projects/ngrx.io/content/guide/component-store/read.md

```diff
@@ -173,6 +173,57 @@ private readonly fetchMoviesData$ = this.select(
 );
 </code-example>
 
+## `selectSignal` method
+
+ComponentStore also provides the `selectSignal` method, which has two signatures. The first signature creates a signal from the provided state projector function, while the second creates a signal by combining provided signals, similar to the select method that combines provided observables.
+
+<code-example header="users.store.ts">
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+
+import { User } from './user.model';
+
+type UsersState = { users: User[]; query: string };
+
+@Injectable()
+export class UsersStore extends ComponentStore&lt;UsersState&gt; {
+  // type: Signal<User[]>
+  readonly users = this.selectSignal((s) => s.users);
+  // type: Signal<string>
+  readonly query = this.selectSignal((s) => s.query);
+  // type: Signal<User[]>
```

Comment:
To display these comments correctly:

```suggestion
  // type: Signal&lt;User[]&gt;
  readonly users = this.selectSignal((s) => s.users);
  // type: Signal&lt;string&gt;
  readonly query = this.selectSignal((s) => s.query);
  // type: Signal&lt;User[]&gt;
```

Currently, the generic type is not displayed: ![selectSignal](https://github.com/ngrx/platform/assets/17877290/ecb9daa4-8868-4e64-b863-ef879a2ae456)